mobile_back_samsung/cpp/eden_mlperf_c_backend/jni/edenai.cc

```diff
@@ -1050,15 +1050,17 @@ void mlperf_backend_clear(mlperf_backend_ptr_t backend_ptr) {
   EdenAI::jni_CloseModel();
   EdenAI::jni_Shutdown();
   libeden_nn_on_direct = nullptr;
+  delete samsung_backend;
 }
 
-Backend ss_backend;
+static Backend* ss_backend;
```

Comment:
I don't understand why this declaration is needed. What is the purpose of this declaration?

Do you mean that ss_backend should hold its address of Backend for its entire life cycle?